code/_onclick/click.dm

```diff
@@ -368,7 +368,7 @@
 	A.CtrlShiftClick(src)
 	return
 
-/mob/proc/ShiftMiddleClickOn(atom/A)
+/mob/proc/MiddleClickOn(atom/A)

```

Comment:
There are no checks at all here. Dead people can point.

Reply:
To be fair, ghosts can point on both TG and Para.

Reply:
It's not broken. Tested it. Alive people can't see ghost points.